sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_LROPolling.md

```diff
@@ -0,0 +1,149 @@
+# Polling the Long Running Operations
+This sample demonstrates how to use polling for the Long Running Operations using the Text Analytics client.
```

Comment:
consider mentioning in the introduction the different ways to poll a LRO... something like
```suggestion
This sample demonstrates the different ways to consume or poll the status of a Text Analytics client Long Running Operation.  It uses the (Analyze Operation)[https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/textanalytics/Azure.AI.TextAnalytics#run-analyze-operation-asynchronously] as an example
```